module/Application/view/partial/main-nav.phtml

```diff
@@ -35,48 +35,13 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
                     </ul>
                 </li>
                 <?php if (null === $this->identity()): ?>
-                    <li class="visible-xs pull-left">
+                    <li class="pull-right">
                         <a href="<?= $this->url('user') ?>">
                             <span class="fas fa-user"></span>
                             <span class="sr-only"><?= $this->translate('Login') ?></span>
+                            <?= $this->translate('Login') ?>
                         </a>
                     </li>
-                    <li class="dropdown pull-left hidden-xs">
```

Comment:
You'd have to check this on a mobile device, but there exists a possibility that there is not enough room to display both the icon and the text (because there is no distinction for different devices).

Reply:
I checked and it fits on any normal sized phone